config/v1/types_infrastructure.go

```diff
@@ -1002,6 +1002,15 @@ type PowerVSPlatformStatus struct {
 	// Note: Currently only single-zone OCP clusters are supported
 	Zone string `json:"zone"`
 
+	// ResourceGroup is the resource group for new IBMCloud resources created for the cluster.
+	// It is created by user to organise the resources in IBMCloud by grouping them
+	// More about resource group https://cloud.ibm.com/docs/account?topic=account-rgs
+	// It will get read by cluster-image-registry-operator to set up COS Instance in IBMCloud
```

Comment:
Currently it's going to get used only by image registry operator.
We are trying to leverage the existing flow of using IBMCOS as backend storage in image registry. It is done for IBMCloud platform.
It is reading configuration from Infrastructure [here](https://github.com/openshift/cluster-image-registry-operator/blob/ef0869116750e5cd10835937c2c77a40d7c0dc31/pkg/storage/ibmcos/ibmcos.go#L139).
 
Want to follow the same logic for PowerVS platform, so we are trying to add ResourceGroup into Infrastructure status like IBMCloud's ResourceGroup [here](https://github.com/openshift/api/blob/6ba31fa438f20d6c822ef47ad3d771309cb5216b/config/v1/types_infrastructure.go#L913) . 

Since we are only trying to leverage the existing flow, have not written design doc. More about this feature can be seen here: https://issues.redhat.com/browse/MULTIARCH-3212

Reply:
Ok, if this is following the pattern of the existing IBM implementation I think this is ok to continue with. Do you know in IBM if anything other than the storage for image registry uses that option?

Reply:
No could not find any other place they are using ResourceGroup field, I could see the usage only in image registry operator.